### ISSUE TITLE
US subsidiary

### ISSUE THREAD

**Issue by Changaco** [liberapay/liberapay.org#45]
Ironically, Liberapay could now benefit from having a legal entity in the US. I thought I had already created an issue about this, but it seems we only have the related one <https://github.com/liberapay/liberapay.com/issues/1354>.

An easy way to do this might be [Stripe Atlas](https://stripe.com/atlas), however we would get some backlash from the community for setting things up that way in a “tax haven” ([Is Delaware a Tax Haven?](https://priceonomics.com/is-delaware-a-tax-haven/)).

Is there anyone in the US who would be willing to help us do this differently?

**Comment by mattbk** [comment]
The benefit of Delaware (as I see it from inside the US) is that they don't require a physical presence, just a mailing address. I've never heard of it as a tax haven, but there must be some benefit for American corporations to be incorporated in Delaware rather than their physical headquarters.

By "do this differently," do you mean acting as the corporate headquarters for Liberapay in the US rather than renting a mailing address in Delaware?

**Comment by Changaco** [comment]
By “differently”, I mean anything other than Stripe Atlas. I'm open to all suggestions on how we could do it and what the pros and cons of those different approaches would be compared to Atlas.

**Comment by DrAlta** [comment]
I'm in Gulfport Mississippi and am willing to help anyway I can.  We are using 501c3GO to incorporate our church. I assume you're going for a Non profit status in the US. do you need any of the other services from Stripe Atlas other than incorporating?

**Comment by Changaco** [comment]
I'm not sure what we need, but I know we don't need a local bank account, because a “Borderless” Transferwise account will be better if we need to transfer funds internationally.

A nonprofit status makes sense insofar as the Liberapay legal entities aren't meant to distribute dividends, but I'm not sure Liberapay fits in any of the [501(c) categories](https://en.wikipedia.org/wiki/501(c)_organization).

**Comment by DrAlta** [comment]
Yeah I would be stretching it to be "relief of the poor or relief of the underprivileged".  Looks like you want to be a public benefit corporation like kickstarter is.

**Comment by DrAlta** [comment]
Well I contacted attorney David G Marmon from 509c3go to see if he could do a consultation on what type of corporation to form.

He replied back to me, "A French nonprofit can start a nonprofit here in the U.S. There isn’t any other type of corporation for them."

**Comment by DrAlta** [comment]
Is there more documentation of what type of corporation Liberapay is under French law and more about it's purpose than what in the bylaws?

**Comment by Changaco** [comment]
Not really, everything is in the bylaws, except the new mission which hasn't been merged yet (#26).
